style.md

```diff
@@ -545,6 +546,102 @@ const (
 // LogToStdout=0, LogToFile=1, LogToRemote=2
 ```
 
+### Use time
+
+Time is complicated. Incorrect assumptions often made about time include the
```

Comment:
I think there's a few parts to the guidance that we'd like:
 - Don't try to do time related calculations yourself, since it's easy to get wrong
 - Use the time related types where possible. E.g., it's supported in flags, YAML, etc.
 - If you need to use int (e.g., interacting with some external system over JSON), then the field should be named with the unit `fooSeconds`.

I'm not sure if they all belong in a single guideline

Reply:
This guidance only covers the first one of those. The second advantage
comes implicitly from using the correct types. Do you want me to
include that as an advantage here?

RE: the third: Instead of making it a time thing, that could be a more
general naming advice around including units in variable names. What do
you think?

Reply:
I think 2nd is worth explicitly calling out, since there are places where you don't necessarily do calculations (e.g., passing in a timeout to some call) but you should still use the type in flags etc.

I like the 3rd being a separate guidance, but we may want to briefly mention or reference here

Reply:
Fair. Added a specific subsection about external systems, recommending time.Time and time.Duration where possible, and int/float with units for time.Duration, strings in RFC 3339 format for time.Time, when not possible.

https://github.com/uber-go/guide/blob/abg/time/style.md#use-timetime-and-timeduration-with-external-systems